DigitalLearningSolutions.Web/Views/LearningContent/_TutorialCard.cshtml

```diff
@@ -0,0 +1,38 @@
+﻿@using DigitalLearningSolutions.Data.Models.TutorialContent
+
+@model TutorialSummary
+
+<div class="searchable-element nhsuk-panel card-with-buttons word-break" id="@Model.TutorialId-card">
+  <details class="nhsuk-details nhsuk-expander nhsuk-u-margin-bottom-0">
+    <summary class="nhsuk-details__summary">
+      <span class="nhsuk-details__summary-text searchable-element-title" id="@Model.TutorialId-name" name="course-name">
+        @Model.TutorialName
+      </span>
+    </summary>
+
+    <div class="nhsuk-details__text">
+
+      @Html.Raw(Model.Objectives)
+      <p class="brands__top-horizontal-dash"></p>

```

Comment:
What is this doing? And does it need to be a separate element?

Reply:
Objectives is stored as a raw HTML page in the database, it comes with multiple tags within itself.

Reply:
Ah sorry, I mean `<p class="brands__top-horizontal-dash"></p>`